### PR TITLE
Reglage discipline

### PR DESCRIPTION
**Pull request by yazidsef**
'reglage de problem de affichage des formations par discipline_id'

### CODE REVIEW

File: src/Model/FormationManager.php [hunk 1]
```diff
@@ -22,4 +24,13 @@ public function selectAllByDiscipline(int $id): array
 
         return $statement = $statement->fetchAll();
     }
+    public function selectOneById(int $id): array|false
```
Comment (Benja-san) [inline src/Model/FormationManager.php:27]: SelectOneByDisciplineId then

File: src/View/discipline/index.html.twig [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <main class="main">
         <section class="section grid">
             <h1 class="formation-title">Nos formations</h1>
-
+ {{ app.request.query.get('id') }}
```
Comment (Benja-san) [inline src/View/discipline/index.html.twig:15]: Remove useless or testing code

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/Model/FormationManager.php (hunk 1 above)
Comment (Benja-san) [reply src/Model/FormationManager.php]: You did not change the method name

File: src/Model/FormationManager.php [hunk 3]
```diff
@@ -22,4 +24,13 @@ public function selectAllByDiscipline(int $id): array
 
         return $statement = $statement->fetchAll();
     }
+    public function selectOneByDisciplineId(int $id): array|false
```
Comment (yazidsef) [inline src/Model/FormationManager.php:27]: @Benja-san  changed !

**Review by Benja-san — APPROVED** [review]

**Review by LaserHuy — APPROVED** [review]
nice

**Pull request merged**